src/commands/ft_create_parser.cc

```diff
@@ -446,14 +555,137 @@ absl::StatusOr<data_model::IndexSchema> ParseFTCreateArgs(
   if (res) {
     return absl::InvalidArgumentError(NotSupportedParamErrorMsg(kFilterParam));
   }
-  VMSDK_RETURN_IF_ERROR(ParseLanguage(itr, index_schema_proto));
-  VMSDK_RETURN_IF_ERROR(ParseScore(itr, index_schema_proto));
-  VMSDK_ASSIGN_OR_RETURN(
-      res, vmsdk::IsParamKeyMatch(kPayloadFieldParam, false, itr));
-  if (res) {
-    return absl::InvalidArgumentError(
-        NotSupportedParamErrorMsg(kPayloadFieldParam));
+  // Parse global text parameters before SCHEMA
+  SchemaGlobTextParams global_text_defaults;
+  // Initialize with defaults for each parse call
+  global_text_defaults.punctuation = ",.<>{}[]\"':;!@#$%^&*()-+=~/\\|";
```

Comment:
`",.<>{}[]\"':;!@#$%^&*()-+=~/\\|"` would be good to store as a static constant instead of just inside this function